### PR TITLE
Add DurationAttributeAccessor

### PR DESCRIPTION
**Pull request by ston1x**

### CODE REVIEW

File: spec/models/duration_attribute_accessor_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    subject do
+      Class.new(ActiveRecord::Base) do
+        self.table_name = 'courses'
+        include NxtSupport::DurationAttributeAccessor
+
+        duration_attribute_accessor :class_duration, :topic_duration, :total_duration
+      end
+    end
+
+    context 'when attributes have ActiveSupport::Duration type' do
```
Comment (nsommer) [inline spec/models/duration_attribute_accessor_spec.rb:24]: I think it would also be good to test the cases of assigning `nil` or an invalid type.

**Review by nsommer — APPROVED** [review]
See comment, otherwise 👍

File: spec/models/duration_attribute_accessor_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      let(:course) do
+        subject.new(
+          class_duration: 1.hour,
+          topic_duration: 1.month,
+          total_duration: 1.year
+        )
+      end
+
+      it 'stores values as ISO8601 strings' do
+        binding.pry
```
Comment (ston1x) [inline spec/models/duration_attribute_accessor_spec.rb]: oops

**Review by AkihikoITOH — APPROVED** [review]
❤️
